### DISCUSSION TITLE
Swipe Up Gesture on Apps

### DISCUSSION THREAD

**Discussion by hustlewithvikram** [LawnchairLauncher/lawnchair#4551] [Ideas]
Well if you check the older lawnchair app you'll find the option to set swipe on apps to open another apps. this is very important for us, like we hide those apps which shouldn't be seen and then to acces them we need a way and its a very useful way to access them. It's not just about accessing the hidden apps, in my case it's a faster way to open apps which are frequently used. 

Additional: You can also add a biometric lock to those apps which will be opened when swiped up on a app.

**Comment by Matth7878** [comment]
Fully agree, swipe up (and even down) on apps or drawer to open a specific app is a must have for quick interaction. It's the main feature preventing me to totally switch from Nova. 😕
